discord/invite.go

```diff
@@ -32,6 +32,16 @@ type Invite struct {
 	InviteMetadata
 }
 
+// URL returns a Discord invite URL linking to the invite.
+func (i Invite) URL() string {
+	return "https://discord.gg/" + i.Code
+}
+
+// LongURL returns a long-form Discord invite URL linking to the invite.
+func (i Invite) LongURL() string {
+	return "https://discord.com/invite/" + i.Code
```

Comment:
Should that const also be used here then? [`message.go`](https://github.com/diamondburned/arikawa/blob/v3/discord/message.go#L118)

Reply:
If you want an `EndpointInvite`, then it should probably be in this file. `BaseEndpoint + "/invite/"` is OK, though.

Reply:
Actually, `EndpointInvite` should probably be in invite.go.

Reply:
Resolving these 2 conversations, since I don't really want additional URL constants in `package discord`. In hindsight, having all URL constants in `discord` would've been the better idea rather than `api`.